src/manage_nsfs/manage_nsfs_cli_responses.js

```diff
@@ -150,25 +170,30 @@ ManageCLIResponse.UpgradeHistory = Object.freeze({
 
 ManageCLIResponse.ConnectionCreated = Object.freeze({
     code: 'ConnectionCreated',
+    message: 'Connection has been successfully created',
```

Comment:
I think that here as well maybe Notification Connection
@alphaprinz probably can help with this response details